Sources/OpenTelemetryInstrumentationSupport/SpanAttribute+EndUser.swift

```diff
@@ -33,20 +33,8 @@ public struct EndUserAttributes: SpanAttributeNamespace {
     public init(attributes: SpanAttributes) {
         self.attributes = attributes
     }
-
-    public subscript<T>(dynamicMember member: KeyPath<NestedAttributes, SpanAttributeKey<T>>) -> SpanAttribute? {
-        get {
-            let key = NestedAttributes.namespace[keyPath: member]
-            return self.attributes[key.name]
-        }
-        set {
-            let key = NestedAttributes.namespace[keyPath: member]
-            self.attributes[key.name] = newValue
-        }
-    }
```

Comment:
this boilerplate does not have to be implemented in all members anymore, just once (by us)